tests/test_lib.py

```diff
@@ -14,6 +14,8 @@
 from dfvfs.path import factory as path_spec_factory
 from dfvfs.resolver import resolver as path_spec_resolver
 
+PROJECT_PATH = os.path.abspath(
```

Comment:
The problem is that some build environments require the paths to the test data to be passed at run-time and defining it as such is not going to work. Let's sync separately about this.

Reply:
I think I understand your concern here. I've refactored so all the paths are defined here for easier overriding. Files need to be updated to read the test_data path at one time anyway, and now all this overriding can take place in one location.

Reply:
Unfortunately the refactored solution is not going to work. The test data path need to be determined at run-time. As I indicated, I can better explain why this is outside this PR.